### DISCUSSION TITLE
Missing "provideAuth(() => getAuth())" in modular and advanced sample?

### DISCUSSION THREAD

**Discussion by RafOXxxx** [angular/angularfire#2996] [Q&A]
I tried to use the modular sample as a template and run into a **NullInjectorError** until I added  `provideAuth(() => getAuth())` to the imports array inside app.module.ts. That made me wonder how the provided samples are functional without it?

**Comment by jamesdaniels** [comment]
In the samples `provideAuth()` is called in the `app.browser.module.ts`. Most the reasons why I split those out should not be necessary as of [Firebase JS SDK 9.1](https://firebase.google.com/support/release-notes/js#version_910_-_september_24_2021)

**Reply by RafOXxxx** [reply]
Thanks for the reply, I totally disregarded that `app.browser.module.ts`.
